src/deploy/functions/runtimes/node/parseTriggers.ts

```diff
@@ -151,9 +153,13 @@ export function addResourcesToBackend(
     }
 
     if (annotation.httpsTrigger) {
-      trigger = {
```

Comment:
This change looks unrelated to AR - it looks ok to me but I just wanted to check that it should be here.

Reply:
Sorry, I was originally writing a bunch of changes at once, but decided to be kinder. This is an intended but unrelated change. It suppresses a warning when deploying GCFv2 HTTPS functions.